src/storage/compactor/src/server.rs

```diff
@@ -124,7 +124,9 @@ pub async fn compactor_serve(
         sstable_store,
         stats: state_store_stats,
         is_share_buffer_compact: false,
-        compaction_executor: Arc::new(CompactionExecutor::new(None)),
+        compaction_executor: Arc::new(CompactionExecutor::new(
+            opts.compaction_worker_threads_number,
```

Comment:
Why not use default core number?

Reply:
To make it configurable. For example to reserve some cores for main runtime.
`compaction_worker_threads_number` is still None by default.

Reply:
OK.